tensorflow_encrypted/convert/register.py

```diff
@@ -0,0 +1,112 @@
+import tensorflow as tf
+
+from typing import Any, Dict, List
+
+
+def register() -> Dict[str, Any]:
+    reg = {
+        'Placeholder': placeholder,
+        'Const': constant,
+        'Conv2D': conv2d,
+        'Relu': relu,
+        'BiasAdd': bias_add,
+        'MaxPool': maxpool,
+        'Shape': shape,
+        'StridedSlice': strided_slice,
+        'Pack': pack,
+        'Reshape': reshape,
+        'MatMul': matmul,
+        'Softmax': softmax,
+    }
+
+    return reg
+
+
+def placeholder(node: Any, inputs: List[str], output_lookup: Dict[str, Any]) -> Any:
+    return tf.placeholder(node.attr["dtype"].type,
```

Comment:
I guess the goal here is to replace those with `tfe` functions?

Reply:
You got it!